tests/FakeItEasy.Specs/NonGenericCreationOptionsSpecs.cs

```diff
@@ -0,0 +1,845 @@
+namespace FakeItEasy.Specs
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.Diagnostics.CodeAnalysis;
+    using System.Linq;
+    using System.Reflection.Emit;
+    using FakeItEasy.Creation;
+    using FakeItEasy.Tests;
+    using FakeItEasy.Tests.TestHelpers;
+    using FluentAssertions;
+    using Xbehave;
+
+    public static class NonGenericCreationOptionsSpecs

```

Comment:
Note: this is an exact copy of `CreationOptionsSpecs`, except that it tests the non-generic `A.Fake`